mlm/app/app.py

```diff
@@ -233,6 +236,11 @@ def start(api, port, name):
     flask_p = multiprocessing.Process(name="flask",
                                       target=Flask(api, port, name))
 
+    def handler(signum, frame):
+        sys.exit(1)
```

Comment:
I suppose we can set `should_stop` event here, which should be checked by task process